### ISSUE TITLE
Issue submitted, I can start work now right?

### ISSUE THREAD

**Issue by iteles** [dwyl/contributing#58]
**Full screen view once I get to step 3:**
<img width="986" alt="screen shot 2017-01-24 at 20 31 14" src="https://cloud.githubusercontent.com/assets/4185328/22265476/216b68da-e274-11e6-9c47-9a26ef58d743.png">

<hr/>

> Amaze! I've submitted an issue, I've starred the repo (coz I'm super nice 😇 )  and I don't have any questions, so I'm gonna go ahead and do the work now to get the feature I need into this open source project because I definitely don't need to scroll down further.

:rotating_light::rotating_light::rotating_light::rotating_light:

There is no indication that there will be further steps.  
I would suggest a sentence at the end of the [submit a new issue step](https://github.com/dwyl/contributing#step-3-submit-the-new-issue) that says something like the following:
`Now one of your team members will need to [validate the need for the story](https://github.com/dwyl/contributing#a-different-team-member-must-verify-the-user-storyissue) before any work can be done on it.`

**Comment by nelsonic** [comment]
@iteles I _hope_ this is addressed by including the diagram: #59

**Comment by iteles** [comment]
Seems to be working for now, no one else has raised this 👍
